deploy/bootstrap.py

```diff
@@ -238,14 +265,36 @@ def get_env_setup(args, config, machine, compiler, mpi, env_type, source_path,
 
     activate_env = f'{source_activation_scripts} && conda activate {env_name}'
 
-    return python, recreate, conda_mpi, activ_suffix, env_suffix, \
-        activ_path, env_path, env_name, activate_env, spack_env
-
-
-def build_conda_env(config, env_type, recreate, mpi, conda_mpi, version,
-                    python, source_path, conda_template_path, conda_base,
-                    env_name, env_path, activate_base, use_local,
-                    local_conda_build, logger, local_mache, update_jigsaw):
+    options['env_name'] = env_name
+    options['python'] = python
+    options['recreate'] = recreate
+    options['conda_mpi'] = conda_mpi
+    options['activ_suffix'] = activ_suffix
+    options['env_suffix'] = env_suffix
+    options['activ_path'] = activ_path
+    options['env_path'] = env_path
```

Comment:
```suggestion
    options['conda_env_path'] = conda_env_path
```
This will also need to be renamed above.